### DISCUSSION TITLE
FIL-RetroPGF-1: Nominating a Project

### DISCUSSION THREAD

**Discussion by kkarrancsu** [filecoin-project/community#689] [RetroPGF]
### Introduction
This thread is the kick-off for the project nominations of the recently announced FIL-RetroPGF-1.

Our vision is to build on the success of Optimism’s RetroPGF efforts and reward work that has positively impacted the Filecoin ecosystem. RetroPGF is an experimental mechanism to realize this goal of “impact = profit.” For more information, see [this announcement](https://filecoin.io/blog/posts/unveiling-fil-retropgf-1-retroactively-funding-filecoin-public-goods/).

Anyone may nominate a project for the first round of FIL-RetroPGF-1! Specific instructions for nominations are listed below.

### Overview of Timelines

- Project nominations: Mar 15 - Mar 31
- Project opt-in window: Apr 8th - Apr 21
- Voting: Apr 22 - May 6

### Scope of this round
Filecoin-RetroPGF-1 will distribute 200k FIL to public goods that support Filecoin in one of the following categories:

- **Infrastructure + Dependencies**: software used to build or deploy the FIL Stack; contributions to protocols or standards upon which the FIL Stack runs; experiments that support future development of the core FIL Stack protocol.
- **Tooling + Utilities**: work that helps builders create applications on Filecoin mainnet, build on the FIL Stack, or use applications built on Filecoin.
- **Education & Outreach**: work to spread awareness and knowledge of how Filecoin works. Both technical and social contributions are welcome!
- **Protocol Research & Development**: work to improve the Filecoin protocol, to include both theoretical contributions as well as implementations and prototypes are in scope.
- **Collective Governance**: Work that impacted Filecoin's governance participants or helped bring new governance participants into the network.
- **Products & End User UX**: Work that helped improve the end-user experience and adoption of Filecoin technology.
- **Other**: if you feel that a project does not fit into any other buckets but is a deserving candidate for FIL-RetroPGF-1 funding, use this category.

_Note: The Filecoin Stack refers to the OSS code powering Filecoin, its dependencies, and related codebases._

### How to Nominate a Project

Anyone can nominate projects that fit into one of the following categories (please click on a category's link to nominate a project in that category):

- [**Infrastructure & Dependencies**](https://github.com/filecoin-project/community/discussions/690)
- [**Tooling & Utilities**](https://github.com/filecoin-project/community/discussions/691)
- [**Education & Outreach**](https://github.com/filecoin-project/community/discussions/692)
- [**Protocol Research & Development**](https://github.com/filecoin-project/community/discussions/693)
- [**Collective Governance**](https://github.com/filecoin-project/community/discussions/694)
- [**Products & End User UX**](https://github.com/filecoin-project/community/discussions/695)
- [**Other**](https://github.com/filecoin-project/community/discussions/696)

Note: The purpose of this thread is to provide feedback on the project nomination process, ask general questions about the FIL-RetroPGF-1 effort, and for other general announcements.  _Please use the specific category's link to nominate a project!_

Things to consider when nominating a project:
- Use your best effort to describe how this project has delivered impact to the Filecoin ecosystem in the previous six months (Oct 2023 - Mar 2024). Use the template provided in each category’s thread when nominating a project.
- Make sure to nominate projects in the relevant category. If you feel that a project spans multiple categories, please choose the one category that you feel it best aligns with. Do not nominate the same project across multiple categories! 
- Note that the organizers of FIL-RetroPGF-1 may choose to reassign a project’s category from its nominated one, if they feel it better aligns elsewhere. 
- Before nominating a project please check if this project has been nominated already, so we can avoid duplication.
RetroPGF is an experiment that Filecoin plans to iterate on regularly. Community feedback is key to helping the process improve with each round. We welcome your notes, reflections, and feedback at any step. Please use this discussion to submit feedback!
- Please use the specific thread for the particular category you are nominating a project in (linked above)!

Project Nominations will close by Mar 31, 2024.

**Comment by jennijuju** [comment]
Checkout @jbenet ’s FUND WHAT MATTERS talk at FDS-4 on why PGF is a critical initiative https://youtu.be/of2c5F2H6Tw?si=vCKsCRw82xb9cQL1 && plans on implementing filecoin rPGF so to support filecoin public good work contributors ! https://youtu.be/of2c5F2H6Tw?si=vCKsCRw82xb9cQL1

**Comment by steven004** [comment]
Awesome! This initiative is commendable, as it not only pledges financial backing for noteworthy projects but also underscores the pivotal role of community engagement and feedback in molding the future of the ecosystem.

Prior to nominating, I'm keen to delve deeper into the specifics of the process, believing that a discussion on the procedural details could be helpful. My current understanding is that project nominations are to be submitted directly within this thread – is my interpretation correct? We need a definition of the subsequent step, specifically the project opt-in process? Is it just a confirmation of the project owner will be happy to get the funding, or anything more? Regarding the final stage of voting, I'm curious about who qualifies as a voter. Additionally, how's the voting rules and the weight each vote carries?

**Reply by tmellan** [reply]
Thanks for the questions and comments!
- Nominations are not submitted directly in this thread. They should be submitted in the category-specific discussion threads. 
- The opt-in process will be an intake form following the Optimism approach and using easy-retro-pgf software. Projects will be contacted to alert them to fill in the application. More details incoming soon.
- Badgeholders will be drawn from developers, SPs, researchers, token holders and other network constituent categories to represent the whole network. The aim is that the badgeholders should have deep domain expertise, alignment with RetroPGF principles and Filecoin mission, bring fresh and insightful perspectives, and be able to make a minimum level of time commitment to review projects. Badgeholders haven’t been selected/confirmed yet, this is in the works. 
- FIL-RetroPGF-1 voting rules will closely follow the Optimism badgeholder manual for the first iteration. But the point is to evolve and improve, so it's possible FIL-RetroPGF ends up diverging in subsequent rounds if we believe a different approach is merited. Badgeholder manual for FIL-RPGF-1 incoming soon.

**Comment by Fatman13** [comment]


**Reply by kkarrancsu** [reply]
Hi @Fatman13 - my apologies for not creating the hyperlinks to the specific threads where project nominations should go. I have added them above, and in each thread, there is a nomination template that we'd like to be used. Would you be able to move this nomination over to the [**_Education & Outreach_**](https://github.com/filecoin-project/community/discussions/692) thread?  If not, I can create it for you.

**Comment by Fatman13** [comment]
**Asia SP Working Group** nomination moved to [here](https://github.com/filecoin-project/community/discussions/694#discussioncomment-8822121) per [comment](https://github.com/filecoin-project/community/discussions/689#discussioncomment-8802416) below.

**Reply by kkarrancsu** [reply]
Hi @Fatman13 
Same comment as above - I apologize for the unclear instructions previously.  Would you be able to move this nomination to the [_**Collective Governance**_](https://github.com/filecoin-project/community/discussions/694) thread and use the template?

**Comment by AlanGreaterheat** [comment]
Exciting initiatives! 

#Education & Outreach#  Nominee DSPA-Asia has played a significant role in assisting CC SPs in their transition as well as in the development of the Filecoin ecosystem.

**Reply by kkarrancsu** [reply]
Hi @AlanGreaterheat - my apologies for not creating the hyperlinks to the specific threads where project nominations should go. I have added them above, and in each thread, there is a nomination template that we'd like to be used. Would you be able to move this nomination over to the [**_Education & Outreach_**](https://github.com/filecoin-project/community/discussions/692) thread?

**Comment by Joss-Hua** [comment]
For **Collective Governance**, i nominate [FIP0021](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0021.md), [FIP0029](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md), ~~[FIP0070](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0070.md), and [FIP0080](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0080.md)~~ initiated and ultimately implemented by the [ForceCommunity](https://www.forcecommunity.io/). FIP is one of the core elements of ecological development, and the above FIPs have influenced the direction of network development from conception to implementation.

**Reply by jennijuju** [reply]
Note that FIP0070 & FIP0080 are not Accepted yet and shipped in an upgrade - would recommend remove them.

**Reply by Joss-Hua** [reply]
thanks for the reminder

**Reply by Fatman13** [reply]
This seems more in the scope of...

> **Protocol Research & Development**: work to improve the Filecoin protocol, to include both theoretical contributions as well as implementations and prototypes are in scope.

**Reply by kkarrancsu** [reply]
Hi @Joss-Hua - my apologies for not creating the hyperlinks to the specific threads where project nominations should go. I have added them above, and in each thread, there is a nomination template that we'd like to be used. Would you be able to move this nomination over to the appropriate thread (perhaps based on your discussion with @jennijuju  and @Fatman13)?

**Reply by jennijuju** [reply]
@Fatman13 I certainly hope protocol R&D is in scope!

but as i read the rules it has

> describe how this project has delivered impact to the Filecoin ecosystem in the previous six months (Oct 2023 - Mar 2024

I'm less certain whether not-finalized FIPs have delivered impact to Fileecoin, but hey - it's up to the community to nominate the project they believe has value and deserves rPGF!

As a core dev and FIP editor, I wanna be cautious this program doesn't create bad incentives where people open "random" FIPs or create OSS PRs so to get rPGF funds - it will waste core dev and implementers time.

See an example here https://bsky.app/profile/web3isgoinggreat.com/post/3kmfslam6ml2m

**Reply by jsoares** [reply]
> As a core dev and FIP editor, I wanna be cautious this program doesn't create bad incentives where people open "random" FIPs or create OSS PRs so to get rPGF funds - it will waste core dev and implementers time.

This isn't a huge risk for programmes with non-mechanical rewards, especially those with permissioned voting. There can always be frivolous PRs/submissions, but it should be obvious from the start that they won't be rewarded so the incentive isn't there.

> describe how this project has delivered impact to the Filecoin ecosystem in the previous six months (Oct 2023 - Mar 2024)

I find the limited "impact window" much harder to interpret... Something that happened two years ago can continue to have an impact now, and often does (this is the case for both discrete events, like an early FIP, or a dependency, like drand). Optimism clearly state that there's no limit to how "old" submissions can be, and while such a limit is not imposed here either, this window teaches away from older contributions.

> I'm less certain whether not-finalized FIPs have delivered impact to Fileecoin

A rejected or draft FIP can also have created impact due to the discussions it prompted, though that's admittedly a less common occurrence. As @jennijuju said, I think the community should nominate what they believe is valuable, and hopefully badgeholders will be given wide latitude in making their determinations.

**Reply by kkarrancsu** [reply]
Hi, @jsoares - thanks for the great questions.  

Badgeholders will have wide latitude in determining how much funding a project receives. Specific guidelines that address questions like these will be provided to the badgeholders before the voting process starts (Apr 22). This information will also be shared publicly so that everyone understands the criteria intended by the FIL-RetroPGF-1 effort.

That being said, we actively want to reward recent impact. This doesn't mean excluding projects that have sustained impact beyond the impact window. It's up to the badgeholders to take into account contextual information.

**Comment by deltazxm** [comment]
For Collective Governance category, i nominate Asia SP working group. This particular branch of SPWG has been holding bi-weekly meetings since time roughly around Filecoin mainnet launch. Group members actively participate in FIP discussions and governance, bridging the chasm between core-devs and SPs.

**Reply by kkarrancsu** [reply]
Hi @deltazxm -my apologies for not creating the hyperlinks to the specific threads where project nominations should go. I have added them above, and in each thread, there is a nomination template that we'd like to be used. Would you be able to move this nomination over to the [**_Collective Governance_**](https://github.com/filecoin-project/community/discussions/694) thread and use the template?

**Comment by Destore2023** [comment]
I also nominate the Asia SP working group for the Collective Governance category.

This particular branch of SPWG has been holding bi-weekly meetings since time roughly around Filecoin mainnet launch. 

Group members actively participate in FIP discussions and governance, bridging the chasm between core-devs and SPs.

**Reply by kkarrancsu** [reply]
Hi @Destore2023  -my apologies for not creating the hyperlinks to the specific threads where project nominations should go. I have added them above, and in each thread, there is a nomination template that we'd like to be used. Would you be able to move this nomination over to the [**_Collective Governance_**](https://github.com/filecoin-project/community/discussions/694) thread and use the template?

**Comment by dannyob** [comment]
Hi @tmellan, @kkarrancsu  ! Who is running this? I suspect we will get queries here at the Foundation about the project, and I'd love to be able to direct people to the correct organization putting this together!

**Reply by kkarrancsu** [reply]
Hey @dannyob - @tmellan and myself will be coordinating the effort, but I don't want to downplay all of the support that we've received thus far.  It would be great if it's possible to direct any questions to this thread, so that we can maintain the open and public nature of the discourse that we hope to foster with the FIL-RetroPGF initiative.

**Reply by tmellan** [reply]
Hey @dannyob, yep as Kiran says we're helping coordinate. It's really a (multi-)community effort, with input already from the good people hailing from the lands of Gitcoin, PL, Optimism, Filecoin community and the Foundation. Would love to work the Foundation more closely too in any capacity --- we're looking for more funders to fund the rounds and more heads to advise and plan and give feedback. Cheers!

**Reply by dannyob** [reply]
Sorry, I should have been clearer: what's the entity that's dispensing the funds? That's often the first question by somebody nominated to receive money,  and can definitely make the subsequent paperwork a lot easier!

**Reply by tmellan** [reply]
The first round will be primarily funded by PL, but other orgs most welcome to join now and in future rounds.

**Comment by RobQuistNL** [comment]
How about miners that provide data and serve retrievals?

**Reply by kkarrancsu** [reply]
Producing software and making it open source that enables this is likely to be considered a public good but serving the data to clients is not.

**Reply by RobQuistNL** [reply]
you mean serving the data to open projects like IPFS.io and storing public datasets? :grimacing:  Like Saturn does? 

Also, I've asked multiple people to open source my management / monitoring interface for Lotus - but the nucleation doesn't seem to help. Where do I submit it?

**Reply by jennijuju** [reply]
👀👀👀

I believe one motivation of rPGF is to be able to incentivize and fund open source project/contribution that has an impact. https://github.com/filecoin-project/community/discussions/689#discussioncomment-8842462 suggested badge holders will evaluate the fund needs when they review the projects.

**Comment by The-Wayvy** [comment]
I'd like to nominate whoever implements Proof of Access with a 10x multiplier for 1x pledge

**Reply by kkarrancsu** [reply]
Hi @The-Wayvy - thanks for your interest in this program. This sounds like a good candidate for proactive funding, but because the product has not yet materialized or provided impact, it would not be eligible for RetroPGF.

**Reply by The-Wayvy** [reply]
Are there any plans for a bounty program?

**Comment by steven004** [comment]
One more question (because it is called RetroGPF): 
Are those projects which already got grants, or funding from investors, especially from PL or FF, still eligible for this funding?

**Reply by kkarrancsu** [reply]
Yes, they should still be eligible for funding, but the amount will be determined by badgeholders according to the guidance provided. The project intake forms will request information about how much funding a particular project has already received. It will be up to the badgeholders to use this information to assess how much to allocate to a specific project.

**Comment by The-Wayvy** [comment]
I'd like to nominate a security audit of RawFIL https://github.com/RawFilecoin/RawFilecoin

**Reply by kkarrancsu** [reply]
Similar to the above, this sounds like it is more applicable for proactive funding.

**Reply by The-Wayvy** [reply]
🤔Hmmm

This is a completed product AFAIK

**Reply by jsoares** [reply]
@The-Wayvy by completed product, do you mean RawFIL or the security audit? Assuming the security audit hasn't been done yet, I think you could nominate RawFIL itself as the completed product, and then the team can (elect to) use any proceeds to pay for a security audit if they so wish.

**Reply by The-Wayvy** [reply]
@jsoares 

Thanks Jorge 🤝

@kkarrancsu 

I'd like to nominate RawFIL.

**Reply by kkarrancsu** [reply]
Hi @The-Wayvy - that sounds good. Can you please add the nomination into the correct category (whichever one you see fit), and fill out the associated template?

Please note that you are welcome to nominate the project, but it needs to have provided impact to the FIlecoin community in the last 6 months. I am not familiar with when RawFil came online, but I think it was recent, but please take that into consideration as well.

**Comment by kelsien** [comment]
@kkarrancsu are the eligibility criteria fixed? I would like to nominate BlockScience but they have done cryptoecon / governance work over a number of years, but the pertinent public goods artifact on the FIP debate is from March, 2023. I would hope the RPG round reflects genuine, long-term contributions if possible

**Reply by kkarrancsu** [reply]
Hi @kelsien - thanks for your interest! 

Even if the artifact was produced in March 2023, if there is a case to be made that it has had **_impact_** in the past six months, it would be eligible for funding.  The impact window is fixed to apply some scope to the program, but there is a separation between impact provided and when the artifact is generated.

**Reply by kelsien** [reply]
Perfect, understand, thanks

**Comment by RobQuistNL** [comment]
Also, can I nominate myself? https://github.com/dukesoft/filecoin-spade-client

**Reply by jennijuju** [reply]
Yes! I think this is a great utility/tooling! Please nominate the project in that category !

**Comment by ArthurWang1255** [comment]
> Filecoin-RetroPGF-1 will distribute 200k FIL to public goods that support Filecoin in one of the following categories

Can projects funded by PL itself be qualified here? Can we check if these projects have been paid for through PL's payroll? 
For example, [this](https://github.com/filecoin-project/community/discussions/690#discussioncomment-8833271), [this](https://github.com/filecoin-project/community/discussions/690#discussioncomment-8854772), [this](https://github.com/filecoin-project/community/discussions/690#discussioncomment-8855175), [this](https://github.com/filecoin-project/community/discussions/690#discussioncomment-8854254)

If these projects above were not already funded by PL, then it make sense to me they are here. Otherwise it looks like a public stunt to give PL subsidiaries money. 
Anyways, we either let everyone vote instead of a permissioned vote OR we exclude PL funded projects.

**Reply by jnthnvctr** [reply]
fwiw my understanding is: 
- anyone can be nominated, its up to teams to accept the nomination to actually apply (and there's a form or something that'll be involved). my 2c has been nominate liberally, as at the very least as a way of showing appreciation for folks who've been doing stuff that's valuable for the ecosystem (whether former PL employees or not). 

- i think in that application process (post nomination), teams will have to disclose if they've received grants / funding previously. from what i know the goal isnt to say X person hasn't received funding before, but to see if theres a gap between impact and amt given (and hopefully public goods funding as a recurring source can help plug the gap). 

- as an aside, not everything being spun out of PL has been "fully funded" - but makes the point above all the more impt because its the gap of impact / funding that i think is relevant. one thing im also hopeful for here (especially for teams who choose to take the second step to apply) is to consider the size of the pool

for this round, i think its just PL who is putting up the money, but hopefully over time if the mechanism works - more people will want to contribute to the matching pool (or create their own versions!).
- im not sure if theyve figured out all the nuances for badgeholders yet (i think theyre borrowing heavily from optimism's set up) - but the reason this pool is relatively small is because theyre trying to work out the kinks. the goal was to get a bunch of different stakeholder groups to be badgeholders (and increase the size of the badgeholder pool as the program ramps).
- one other thing they said at the FDS in Denver is that badgeholders cannot vote on anything they have financial upside in
- i dont know how closely theyve looked into quadratic funding, but hopefully we can incorporate some elements where the community is directly voting with some skin in the game

**Reply by jennijuju** [reply]
You linked some of my nominations, which I nominated those projects that  I believe are critical to the filecoin network and  its user - they are public good work (OSS, free to use) that have created/are creating impacts. It is true that over the years, Protocol Labs had funded individuals or groups to maintain and contribute to some of the projects, yet there were also many other OSS contributors allocated their time into these projects. It is important for these projects to have continuous incentives for OSS developers to  contribute to their maintenance and development, incentives can be in many form (1) proactive funding like grants (2) retroactive funding like rPGF (3) donation, and I might be missing some.

It will be great that many organizations and individuals will contribute to Filecoin public good fund pool, and those funds can be distributed to the **public good work** that is important to Filecoin’s development.

**Reply by jennijuju** [reply]
Without reading into too much to your message, imho many of the projects nominated by the community can use multiple funding sources to sustain their projects, hopefully extend their resources to further build better softwares, toolings and services for filecoin.

**Reply by ArthurWang1255** [reply]
I agree that such funding projects are good for the construction of filecoin.

What I think is that in all projects, when the builders are employees of PL, the employees should seek rewards from the capitalists who exploited their labor surplus instead of from the community campaign(does this count?), because there is a labor agreement between the employee and the company.

Of course, anyone who has been nucleated/left from PL is eligible to participate in the second round of applications after there is clear evidence to prove that the labor relationship with PL has been completely terminated.

I strongly do not recommend retroactive actions to PL related projects without clear standards to confirm which projects have been already funded through the labor agreements.
Especially when Filecoin is notorious for being kleptocratic in the industry.

**Reply by wheresaddie** [reply]
does anyone know who the badge holders are who are voting? and how the votes will be weighted or not?

**Reply by kkarrancsu** [reply]
We'll be releasing some documents soon after nominations close that provide further details on how badgeholders were selected and the guidelines they should be following when allocating funding to each project.

**Comment by elizabeth-griffiths** [comment]
Can you help clarify how we are defining a "project"? 
For example, there might be a product that aligns well with Products & End User UX, and that product might be used in an education initiative "project". If the same team leads the Product and spearheaded the education initiative, can we consider them seperate "projects" for the sake of RPGF?

**Reply by tmellan** [reply]
Sounds like a single project that fits multiple labels, more than two independent projects. If this is the case, picking the single category in which most impact was delivered is likely best.

**Comment by cbtan1** [comment]
**Nomination**
[Chymia](https://chymia.finance/ ) - The first LST-backed and overcollateralized stablecoin on FVM

**A description of the positive impact of this project for this specific category over the past six months (Oct 2023 - Mar 2024)**

**Introduction**
Chymia is Filecoin’s first LST-backed and overcollateralized stablecoin protocol. Its mission is to empower the Filecoin community to maximize the efficiency and liquidity of their tokens in a permissionless environment.

On Chymia, users deposit FVM’s tokens, such as FIL or liquid staking tokens, and borrow cUSD, a Filecoin native stablecoin that is not reliant on non-crypto assets. The protocol is designed to ensure that cUSD is overcollateralized, and can be used at any time to redeem deposited collateral at face value. 

Being able to borrow cUSD using idle tokens opens up unlimited possibilities for Filecoin users to increase yields and manage risks. Having a decentralized and overcollateralized stablecoin also opens the door for FVM builders to start deploying wide-ranging dApps that can benefit the entire Filecoin community.

**Some examples of direct positive impact**

- LST holders can use their LSTs to borrow cUSD and purchase more LSTs, thereby increasing the yields that they can earn on their native FIL tokens
- Since cUSD is non-custodial and permissionless, FIL holders can manage risks and capital more efficiently 
- cUSD gives FIL holders the freedom and flexibility to hold onto their FIL while using other dApps
   - Swap for other tokens
   - Deploy in liquidity pools
   - Deploy in Chymia’s Stability Pool and play a role in cUSD’s overcollateralization and decentralization

We have been building Chymia for the past few months and the product is currently on Calibration testnet. We aim to launch on Mainnet by late April 2024. 

**A link to the project’s GitHub or Twitter**
[Gitbook](https://docs.chymia.finance/ )
[Twitter](https://twitter.com/ChymiaFinance )
[Discord](https://www.chymia.finance/discord)

**Contact info for the project or project lead**
cb@chymia.finance

**Reply by cbtan1** [reply]
have posted this to https://github.com/filecoin-project/community/discussions/695.

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This week marks an important milestone for the FIL-RetroPGF-1 effort.  Nominations for projects closed on Mar 31, 2024.  We received over 150 nominations for projects across the various categories that were eligible for FIL-RetroPGF-1 funding. 

Thank you to the community for taking the time and effort to put these nominations together. The projects nominated reflect all of the incredible work and progress that the Filecoin community have made together.

As FIL-RetroPGF-1 organizers, our next step is to review the nominations and select eligible projects to apply formally to be considered for FIL-RetroPGF-1 funding. We will be using the criteria outlined [in this document](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine whether a nominated project should be asked to apply.  Nominated projects will be informed soon, and applications are currently slated to open on Apr 8th.

Thank you, stay tuned, and check back to this page and main our [RetroPGF landing page](https://fil-retropgf.notion.site/FIL-RetroPGF-4b6f5358440043c8bb1bf53f0297541e) for future developments!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

The FIL-RetroPGF organizers have spent the last week reviewing the nominations submitted in each of the eligible categories. We used the criteria outlined [here](https://fil-retropgf.notion.site/Nomination-Review-Process-32b62a5ea2cd477f83692cbc233edfda) to determine which projects would be invited to apply in this round. Our assessment of eligibility is based on our best interpretation of the information provided in the nomination but tended to bias towards inclusivity, deferring judgement to badgeholders as much as possible. 

The following [page](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) contains a list of projects that are being invited to apply to be eligible for FIL-RetroPGF-1 funding.

Applications are open from 8 Apr 2024 to 22 Apr 2024. We encourage you to submit your application [here](https://www.fil-retropgf.io/) as soon as possible!

Best regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

This is a gentle reminder that the application submission deadline is fast approaching.  If you are a project invited to apply and have not already done so, please submit your applications [here](https://www.fil-retropgf.io/) by Apr 22.

We have updated the [Round 1 - Invited Applications](https://fil-retropgf.notion.site/Round-1-Invited-Applications-3fe95da0c9da4aeba9c6420d1efd0bd6) page to include a POC. Additionally, we have published [the following guidelines](https://fil-retropgf.notion.site/Project-Application-Guidelines-4cf9df76d0f74da9b02cd33f09cbd5b4) on how to approach the application process and included an FAQ at the bottom.

Best Regards,
FIL-RetroPGF Organizers

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

We have extended the deadline to submit your applications. If you haven’t already done so, please submit your application [here](http://fil-retropgf.io/) by 12PM UTC Apr 24, 2024. There will be no further extensions.

Best regards.

**Comment by kkarrancsu** [comment]
Dear Filecoin community,

Thank you for your continued patience. The organizers have finished tallying the results and computing the allocations for each project, according to the scoring rules that were previously set forth. Please visit the following resources for the results:

- [FIL-RetroPGF-1 Results overview](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628)
- [Results Spreadsheet](https://docs.google.com/spreadsheets/d/15tQZU4yfzCCso_dd5NGXfSy6XB2pmrEZzZrG_TEIKyY/edit#gid=1610641018)
- [Deep dive into the results.](https://medium.com/p/7e5a0bcdba08)

We will contact project POCs in the coming days to coordinate distribution. A huge thank you to the entire Filecoin ecosystem for coming together to make this first round successful. Be on the lookout for Round 2!

**Comment by kkarrancsu** [comment]
Dear Filecoin Community,

Thank you for your continued patience. We are now in the next phase of FIL-RetroPGF-1—funds distribution. The [Open Impact Foundation](https://openimpact.foundation/) will coordinate distribution and contact projects [that have been allocated funds](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628?pvs=74) from the FIL-RetroPGF-1 process.

Please be on the lookout in your inbox for emails coming from hello@openimpact.foundation

Best Regards,
